### PR TITLE
Replace log calls

### PR DESCRIPTION
**Pull request by lognaturel**
Now that we are using ✨Timber✨, we should replace all `Log` calls. This is a start and we should do this across the codebase.

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/spatial/OsmMBTileSource.java [hunk 1]
```diff
@@ -112,7 +113,7 @@ public static OsmMBTileSource createFromFile(File file) {
 
             Bitmap bitmap = BitmapFactory.decodeStream(is);
             tileSize = bitmap.getHeight();
-            Log.w(t, String.format("Found a tile size of %d", tileSize));
+            Timber.w(t, String.format("Found a tile size of %d", tileSize));
```
Comment (yanokwa) [inline collect_app/src/main/java/org/odk/collect/android/spatial/OsmMBTileSource.java]: I think this should be `Timber.w("Found a tile size of %d", tileSize);`

File: collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java [hunk 2]
```diff
@@ -1368,17 +1358,15 @@ private void showNextView() {
                     showView(next, AnimationType.RIGHT);
                     break;
                 case FormEntryController.EVENT_REPEAT_JUNCTURE:
-                    Log.i(t, "repeat juncture: "
-                            + formController.getFormIndex().getReference());
+                    Timber.i("repeat juncture: %s", formController.getFormIndex().getReference());
```
Comment (yanokwa) [inline collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java]: Title case please

File: collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java [hunk 3]
```diff
@@ -649,11 +647,9 @@ protected void onActivityResult(int requestCode, int resultCode,
 
                 File nf = new File(s);
                 if (!fi.renameTo(nf)) {
-                    Log.e(t, "Failed to rename " + fi.getAbsolutePath());
+                    Timber.e("Failed to rename %s", fi.getAbsolutePath());
                 } else {
-                    Log.i(t,
-                            "renamed " + fi.getAbsolutePath() + " to "
-                                    + nf.getAbsolutePath());
+                    Timber.i("renamed %s to %s", fi.getAbsolutePath(), nf.getAbsolutePath());
```
Comment (yanokwa) [inline collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java]: Title case

File: collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java [hunk 4]
```diff
@@ -297,12 +296,12 @@ public void onClick(View v) {
             }
             if (savedInstanceState.containsKey(KEY_XPATH)) {
                 startingXPath = savedInstanceState.getString(KEY_XPATH);
-                Log.i(t, "startingXPath is: " + startingXPath);
+                Timber.i("startingXPath is: %s", startingXPath);
```
Comment (yanokwa) [inline collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java:299]: Title case

File: collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java [hunk 5]
```diff
@@ -297,12 +296,12 @@ public void onClick(View v) {
...
             }
             if (savedInstanceState.containsKey(KEY_XPATH_WAITING_FOR_DATA)) {
                 waitingXPath = savedInstanceState
                         .getString(KEY_XPATH_WAITING_FOR_DATA);
-                Log.i(t, "waitingXPath is: " + waitingXPath);
+                Timber.i("waitingXPath is: %s", waitingXPath);
```
Comment (yanokwa) [inline collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java:304]: Title case

File: collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java (hunk 4 above)
Comment (lognaturel) [reply collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java:299]: That's the var name so I think not

**Pull request merged**